requirements.txt

```diff
@@ -73,6 +70,7 @@ unidecode==0.4.19         # via python-slugify
 urllib3==1.22             # via elasticsearch
 venusian==1.0
 vine==1.1.4               # via amqp
+webencodings==0.5.1       # via html5lib
```

Comment:
I don't know why this was missed previously, but the [version of html5lib we use](https://github.com/html5lib/html5lib-python/releases/tag/0.999999999) definitely has a [dependency on webencodings](https://github.com/html5lib/html5lib-python/blob/6a73efa01754253605284b5a5688de3961b120fa/setup.py#L62).

Reply:
I was wondering the same thing!